extra/linux/org.alacritty.Alacritty.appdata.xml

```diff
@@ -4,8 +4,8 @@
   <id>org.alacritty.Alacritty</id>
   <!-- Translators: The application name -->
   <name>Alacritty</name>
-  <project_license>APACHE-2.0</project_license>
-  <metadata_license>APACHE-2.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
You can't just change the metadata license. The existing text is not available under CC0.

Reply:
I can change it back to "Apache-2.0", but it seems to be an invalid metadata license and causes both validators I have tried (appstreamcli and appstream-util) to mark the validation as failed.

Reply:
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-metadata_license

> While copyleft licenses like the GPL are great for code, it is not feasible to test every copyleft license for mutual compatibility and compliance when combining metainfo metadata with other data into one larger assembly fully automatically.

> Currently, the following licenses have been reviewed and can be used as metadata licenses:
```
FSFAP
MIT
0BSD
CC0-1.0
CC-BY-3.0
CC-BY-4.0
CC-BY-SA-3.0
CC-BY-SA-4.0
GFDL-1.1
GFDL-1.2
GFDL-1.3
BSL-1.0
FTL
FSFUL
```

Reply:
I guess one that could work here is MIT, since it's all permissive and compatible with APACHE, given that we don't use copyleft, requiring permission of the author is not needed, thus even cc may work.

Reply:
> given that we don't use copyleft, requiring permission of the author is not needed

I'm not sure what you mean by this. No matter what, we'd require permission by the author if we're changing the license.

Reply:
The author of the [original PR](https://github.com/alacritty/alacritty/pull/2419) sadly does not seem to be on GitHub anymore and their e-mail account is no longer valid: `derpmatt <49352270+derpmatt@users.noreply.github.com>`

Reply:
Yes, so the only option is rewriting the manifest from scratch, which I cannot do because I've been tainted. So we cannot change the license.

Reply:
The current text was mostly written by @chrisduerr according to git blame, with the exception on how markup is placed. I'm not sure there's a need to contact original author, if most of the lines are changed and only _standard defined markup_ was left in place, which is there just because of standard and you can not really change it. One can update `keywords`.

The copyright at the top also a bit doesn't really matter, since there's no entity like alacritty project contributors, since there's no CONTRIBUTORS.md or other entity to contact/etc. Though, on github one can say that it's contributors that popup from the contributors menu, but it was a topic of the past as well iirc.

Also, is this field mandatory?

Reply:
Not an authoritative answer but others have had the exact same question here: https://opensource.stackexchange.com/questions/11531/if-i-fork-a-lib-that-is-mit-apache-2-0-dual-license-can-i-change-it-to-cc0-fo

Reply:
And here's a thread on why Rust itself is duel licensed, in case you were wondering (as I was): https://internals.rust-lang.org/t/rationale-of-apache-dual-licensing/8952 since we do the same thing.

Reply:
we're not dual licensed, that's the thing if we were, that was not a problem.

Reply:
What do you call the two license files at the root of the repo then, @kchibisov?

Reply:
As I understand it Apache and MIT are essentially compatible. So we should be able to choose whichever one we want here. If it makes matters simpler, I don't see the problem with changing this to MIT, but I'm not a lawyer.

Reply:
they just lay there, but the application of them is explicitly specified. this particular file is apache licensed anyway.

Reply:
The issue is that you still need to ask actually, even if they are compatible.

Reply:
I don't see what you're getting at. This file has a nebulous "Copyright 2016-2022 Joe Wilm, The Alacritty Project Contributors" at the top, but otherwise only contains a `project_license` and `metadata_license`, which as far as I can tell should be one of "Apache-2.0" or "MIT", since those are the licenses we use for the project (from the root of the repo).

Reply:
alacritty is apache-2.0 only software, and always been. MIT is used by 2 small crates and it lays in project root only because it's simple to link to it.

Reply:
As far as asking. Who do you mean to ask? Joe Wilm conceded control of this repository to us back in like 2018.

Reply:
> alacritty is apache-2.0 only software, and always been. MIT is used by 2 small crates and it lays in project root only because it's simple to link to it.

https://github.com/alacritty/alacritty/commit/89f4bdb770fad303a122172c60b5280e5061aeaf#commitcomment-109118596

Reply:
The person who originally submitted it, which is no longer on github and has no email, that one should be asked, as it was outlined above. we can not contant them, so nothing could be done unless the file is rewritten.

Reply:
Well, I can change the metadata license back to "Apache-2.0" (case sensitive correct version of the original license) for now. It is incorrect and will cause some AppStream parsers to ignore the metadata, but at least some other will hopefully work.

Reply:
OK, FWIW. I was wrong about the dual license I guess. The project includes an MIT license for some subcomponents, but the authoritative top level license is defined in the Cargo.toml in reference to Apache-2.0.